### DISCUSSION TITLE
​ About Customise direct link

### DISCUSSION THREAD

**Discussion by zxfccmm4** [spencerwooo/onedrive-vercel-index#442] [Q&A] [answered]
Thank you for developing such an excellent project, I have followed the tutorial to deploy my own network disk, but found there is a problem. Just in the example page there is a​ **Customise direct link**, but I don't have it in my deployed page. Attach my network disk link: [Https://shujin.fun](Https://shujin.fun)
How to solve this?
Thank you so much
![image](https://user-images.githubusercontent.com/17553115/154014626-31d58ba9-26cf-4b3d-8078-1d512de3eeba.png)![image](https://user-images.githubusercontent.com/17553115/154014766-1537856f-2e25-464b-81ba-99eac8324b25.png)

**Comment by spencerwooo** [comment, marked as answer]
You sure you’re on the latest commit version?

**Reply by zxfccmm4** [reply]
Thanks for your reply, I have used the new version and it has been solved successfully. Thanks again
